libtenzir/src/execution_node.cpp

```diff
@@ -181,6 +181,7 @@ class exec_node_control_plane final : public operator_control_plane {
       diagnostic::error("{}", error)
         .note("from `{}`", state_.op->name())
         .emit(diagnostics());
+      return;
```

Comment:
This can go when you rebase, I rewrote the abort and diagnostic-emitting functions in the blocking operators fix.